Alumni_Shuffler_Official.ipynb

```diff
@@ -21,7 +21,7 @@
         "colab_type": "text"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/nneibaue/alumni_shuffler/blob/master/Alumni_Shuffler_Official.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/nneibaue/alumni_shuffler/blob/nate_testing/Alumni_Shuffler_Official.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
Might want to switch this hyperlink back to master once the merge goes through

Reply:
Definitely agree, but I want to keep it the same in this branch until we talk to Lynde and Betsy about switching to master